app/controllers/vm_common.rb

```diff
@@ -127,6 +127,10 @@ def websocket_use_ssl?
   private :websocket_use_ssl?
 
   def launch_html5_console
+    # TODO: The override solution is just partial!
+    # After the WebSocket proxying/multiplexing is done on port 443, use the following line as an override:
+    # override_content_security_policy_directives(:connect_src => ["'self'", "wss://#{request.env['SERVER_NAME']}"]
+    override_content_security_policy_directives(:connect_src => ['*'])
```

Comment:
NOTE:
we need to use * to support multiple ports
Change when we use websocket multiplexing